tests/api/controllers/supervisor_import_robot_node/supervisor_import_robot_node.c

```diff
@@ -40,6 +41,20 @@ int main(int argc, char **argv) {
   position = wb_supervisor_node_get_position(robot);
   ts_assert_boolean_equal(position[2] < (previousZ - 0.02), "The controller of the imported robot is not started correctly");
 
+  // import device after controller start
+  WbNodeRef self_node = wb_supervisor_node_get_self();
+  WbFieldRef self_children = wb_supervisor_node_get_field(self_node, "children");
+  wb_supervisor_field_import_mf_node_from_string(self_children, 0, "Camera { name \"imported camera\"}");
+
+  WbDeviceTag camera = wb_robot_get_device("imported camera");
+  ts_assert_int_not_equal(camera, 0, "Camera imported during controller execution not found.");
+  wb_camera_enable(camera, TIME_STEP);
+
+  wb_robot_step(TIME_STEP);
+
+  const unsigned char *image = wb_camera_get_image(camera);
+  ts_assert_boolean_not_equal(image == NULL, "Camera image is NULL.");
```

Comment:
```suggestion
  ts_assert_pointer_not_null(image, "Camera image is NULL.");
```

Reply:
Your suggestion throws errors on compilation:
```
supervisor_import_robot_node.c:56:30: warning: passing argument 1 of ‘ts_assert_pointer_not_null’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
```

We could add a new function taking constant pointer into account, but it works perfectly fine with the `ts_assert_boolean_*` existing function so for me it would be useless.

Reply:
Ah ok you right, in that case, lets keep the `ts_assert_boolean_*` indeed.